open_academy/demo/category.xml

```diff
@@ -0,0 +1,10 @@
+<odoo>
+    <record id="res_partner_category_15" model="res.partner.category">
+        <field name="name">Teacher / level 1</field>
+        <field name="color" eval="11"/>
```

Comment:
Is this working? please change it to RGB color.

https://www.rapidtables.com/web/color/RGB_Color.html